### ISSUE TITLE
LxssManager hangs on latest insider build

### ISSUE THREAD

**Issue by Steveice10** [Microsoft/WSL#2933]
Version: Microsoft Windows [Version 10.0.17093.1000]

After updating to the latest insider build, trying to open Ubuntu bash resulted in an empty terminal with a blinking cursor. My fan also started running constantly, so I checked Task Manager and found that  LxssManager was constantly running with 20-30% CPU usage and no disk usage. I tried stopping LxssManager, but it would not respond to stop commands, so I then tried reinstalling the WSL feature. This worked to open bash once, but after restarting it went back to having the same problem again.

I've attached event logs from when LxssManager first stops working, if they're of any help:
[eventlogs.zip](https://github.com/Microsoft/WSL/files/1713683/eventlogs.zip)

**Comment by Aigeec** [comment]
Version: Microsoft Windows [Version 10.0.17093.1000]

I see similar symptoms. For me LxssManager is stuck stopping. When I first access bash it tells me I don't have distro installed. Subsequently it will just hang when the bash window is opened.

After a reboot bash opens correctly.

Please see ETL logs taken while bash was hung.

[etl.zip](https://github.com/Microsoft/WSL/files/1714673/etl.zip)

**Comment by benhillis** [comment]
I recently found a issue that could explain these hangs. The fix is checked in a should be available soon.

**Comment by Steveice10** [comment]
That's what I thought it was, but Task Manager reported no filesystem activity throughout the entire time I waited. Also, as I mentioned, I was able to successfully start bash once if I disabled and re-enabled the WSL feature, but it would hang again after rebooting.

I did update to yesterday's insider build and try again, though. This time, trying to open bash resulted in:
```
Installing, this may take a few minutes...
It appears this distro's installation has become corrupted.
Please try uninstalling and reinstalling the app.
Installation Failed!
Error: 0x800700b7
```

I uninstalled Ubuntu, reinstalled from the Microsoft Store, and it seems to be working fine now.
